domains/transfer-agreements/TransferAgreements.API/API/Options/OtlpOptions.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace API.Options;
+
+public class OtlpOptions
+{
+    public const string Prefix = "Otlp";
+
+    [Required]
+    public Uri ReceiverEndpoint { get; init; } = null!;
```

Comment:
This Option-class is a bit different than the others as they have a getter and a setter. Not sure if it is okay to have differences here, but wanted to point it out.

Reply:
Make others required and init